.travis.yml

```diff
@@ -2,35 +2,31 @@ language: php
 
 sudo: false
 
+env:
+  COMPOSER_FLAGS="--no-interaction --prefer-source --dev"
+
 php:
-  - 5.6
-  - 7.0
-  - 7.1
   - 7.2
   - 7.3
   - 7.4snapshot
```

Comment:
I think snapshot is not needed anymore? Am I right?

Reply:
It was needed 4 days ago

Reply:
Nice!